aspnetcore/fundamentals/localization-extensibility.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Localization Extensibility
+author: hishamco
+description: Learn how to extend the localization APIs in ASP.NET Core apps.
+ms.author: riande
+ms.date: 03/27/2019
+uid: fundamentals/localization-extensibility
+---
+# Localization Extensibility
+
+By [Hisham Bin Ateya](https://github.com/hishamco)
+
+This article lists the extensibility points on the localization APIs and provides instructions on how to extend ASP.NET Core app localization.
+
+## Extensible Points in Localization APIs
+
+ASP.NET Core localization APIs are build from ground up to be extensible, this will allow the developers to customize the localization according their needs. For instance [OrchardCore](https://github.com/orchardCMS/OrchardCore/) takes the initiative to build a `POStringLocalizer` which described in detail [Portable Object localization](fundamentals/portable-object-localization) to use `PO` files to store the localization resources.
+
+This article lists two extensibility points that localization APIs have, including localization culture providers and localization resources baking store.
```

Comment:
```suggestion
This article lists two extensibility points that localization APIs have: the localization culture provider and localization resources baking store.
```

What do you mean by "localization resources baking store"?

Reply:
The underlying storage that contains the localization resources